README.md

```diff
@@ -277,6 +277,10 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Publishing code coverage to CodeClimate](https://github.com/paambaati/codeclimate-action)
 - [Update repository go report card](https://github.com/creekorful/goreportcard-action)
 
+#### Code Quality
+
+- [Run Gofmt to check Golang code formatting](https://github.com/Jerome1337/gofmt-action)
```

Comment:
Again, this changes the code as opposed to static analysis.

Reply:
Also, please make sure all of your titles are Title Cased: https://github.com/sdras/awesome-actions/blob/master/contributing.md

Reply:
@jessicalostinspace I've made some changes

Reply:
This still isn't title cased